packages/scriptural-react/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@scriptural/react",
+  "version": "0.0.17",
+  "description": "A React-based Bible editor component library built on top of Lexical, providing components, hooks, and utilities for creating scripture editing applications",
+  "main": "./index.js",
+  "module": "./index.js",
+  "types": "./index.d.ts",
+  "type": "module",
+  "sideEffects": false,
+  "keywords": [
+    "bible",
+    "scripture",
+    "editor",
+    "react",
+    "lexical",
+    "usfm",
+    "usj",
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BiblioNexus-Foundation/scripture-editors.git",
+    "directory": "packages/scriptural-react"
+  },
+  "bugs": {
+    "url": "https://github.com/BiblioNexus-Foundation/scripture-editors/issues"
+  },
+  "homepage": "https://github.com/BiblioNexus-Foundation/scripture-editors/tree/main/packages/scriptural-react#readme",
+  "scripts": {
+    "build": "vite build",
+    "test": "vitest run",
+    "test:watch": "vitest"
+  },
+  "peerDependencies": {
+    "react": ">=18.3.1",
+    "react-dom": ">=18.3.1",
+    "lexical": "^0.21.0",
+    "@lexical/react": "^0.21.0"
```

Comment:
Maybe I'm wrong but aren't both of these lexical items actually `dependencies` rather than `peerDependencies`?

Reply:
Not for this version of the package. Since I am allowing the user to add their own lexical extensions if needed, and they will probably need some lexical Types or functions that would not be bundled with this package. First it was a dependency but had to change it because of the above.